firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -5766,6 +5766,14 @@ extension String {
             tableName: "Toolbar",
             value: "New Tab",
             comment: "Accessibility label for the new tab button that can be displayed in the navigation or address toolbar.")
+        public struct tabToolbarLongPressActionsMenu {
+            public static let closeThisTabButton = MZLocalizedString(
+                key: "Toolbar.Tab.CloseThisTab.Button.V130",
+                tableName: "Toolbar",
+                value: "Close This Tab",
+                comment: "Label for button on action sheet, accessed via long pressing tab toolbar button, that closes the current tab when pressed"
+            )
+        }
```

Comment:
Should we create a separate string PR since our string freeze is coming up?
Have a couple of comments:
* `v130` instead `V130`
* Capitalize struct name
* Can we confirm if we should use `Close this Tab`? I thought we were going towards sentence casing, but not too sure if this is cross project.

Hmm, I do agree that our static let should be lowercase, but not consistent with our other variable names. @adudenamedruby any reason why we keep it capitalized?

Reply:
If this lands in `main` before Tuesday, we shouldn't worry about a separate PR. If it's not in by tuesday, yes, separate PR.

re capitalization on strings.... this is personal peeve of mine in the codebase. Yes, static let should be lowercase, but our old strings were capitalized before I arrived and it makes no sense, but such is the life our Strings files lives. We should stay consistent with the rest of the file, unless we want to make sweeping changes, in case we should put our foot down, and start making tickets to for contributors to change things all over. lol

Reply:
will break this out into a separate PR and consult content regarding casing!

Reply:
String PR: #21331

Reply:
great! thank you~